ote/src/clj/ote/tasks/pre_notices.clj

```diff
@@ -179,10 +205,16 @@
          (blue-button (str (environment/base-url) "#/authority-pre-notices") "Siirry NAP:iin tarkastelemaan lomakeilmoituksia")])
 
       (when (seq detected-changes)
-        [:div
+        [:div {:style "background-color:#FFFFFF"}
```

Comment:
Minor remark: should background color `#FFFFFF` be put into a symbol because it's referred twice?